jinja2/includes/polyfill_scripts.html

```diff
@@ -0,0 +1,14 @@
+<script>
+  window.fetch ||
+    document.write(
+      '<script src="https://unpkg.com/unfetch/polyfill"><\/script>'
+    );
+</script>
+<script>
+  // Only include the fat polyfill for browsers that seem to not have
+  // certain JS features. E.g. Firefox 38.
+  Array.prototype.includes ||
```

Comment:
I think I picked this test because it was the first error I got. We just need *something* that gives us a decent indication that the browser is old. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes